README.md

```diff
@@ -396,6 +396,11 @@ There are also 3 networks:
 3. `aws_discover`: network allowing `iot_pub` container to discover Greengrass device. This is
    not required once discovery is complete.
 
+## O3DE Sample
+Open 3D Engine (O3DE) , a community-driven, open-source simulator that provides the high-fidelity realistic rendering necessary for robotic simulations. O3DE is a powerful 3D engine capable of creating and running realistic 3D worlds, making it an ideal tool for gaming and robotic applications.
+
+We integrate ROS2 robots as client devices with the AWS Greengrass V2 (GGv2) server. We enhanced the solution to utilize the robot navigation data using O3DE simulation and navigation stack containers as client devices. By leveraging FastDDS as the ROS protocol, the O3DE and navigation containers can publish data that is received by an IoT Publisher component. The centralized GGv2 gateway servers facilitate client device connectivity, authentication, and provide an MQTT broker. This Publisher can then use Greengrass cloud discovery to locate and authenticate with Greengrass, establishing a secure connection to IoT Core. Using RViz, you can control the robot navigation, which will publish constant linear and angular velocity data to the IoT Core via the MQTT broker. Once your data is available on AWS IoT Core, you can leverage this IoT connectivity to unlock various use cases for improving your robotic system. 
+
```

Comment:
Please discuss how this example extends the previous example, why it's useful (you're using simulated data instead of the random mock data, which is more realistic), and how to run it (build RobotNavVacuum sample, EC2 instance with drivers installed, docker compose commands etc) (EC2 drivers you can just point to docs)

Reply:
Good instructions. Minor typos: vacumm should be vacuum. There are 5 containers to run - "Run all four containers" -> "Run all five containers".  The most important change is to use `docker compose -f docker-compose.o3de.yml up` rather than `docker compose up`. This is true of the workshop as well.